client/src/i18n/fr/creditor_group.json

```diff
@@ -1,4 +1,5 @@
 {"CREDITOR_GROUP":{"BACK":"Retour vers la liste des groupes crediteurs",
 "CREATE":"Creation d'un groupe crediteur",
 "UPDATE":"Mise à jour du groupe crediteur",
-"FAILURE_DELETE":"Echec de suppression du groupe crediteur"}}
\ No newline at end of file
+"FAILURE_DELETE":"Echec de suppression du groupe crediteur",
+"NO_CREDITOR_GROU_ENRED_YET": "Il y a aucun group crediteur enregistré jusque là"}}
```

Comment:
Go ahead and name this `NO_CREDITOR_GROUP` - it will be easier to read what the key means.